app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -119,4 +119,23 @@ def remove_disks(disks, vm)
       disks.each { |disk_id| service.attachment_service(disk_id).remove }
     end
   end
+
+  def vm_migrate(vm, options = {})
```

Comment:
yes, this how our api currently works. `VmOrTemaplate#migrate` calls `VmOrTemaplate#raw_migrate` which calls `Ems#vm_migrate` - so this should work